src/pybind/mgr/diskprediction/common/client_pb2.py

```diff
@@ -0,0 +1,1775 @@
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
+# source: mainServer.proto
```

Comment:
@tchaikov If we add mailServer.proto in the source tree, so We need add convertor script for proto to py code in the ceph-mgr deploy script. Do we need do this?

Reply:
@hsiang41 i am not sure if i am following you. could you define "ceph-mgr deploy script"? is it the `src/pybind/mgr/diskprediction/CMakeLists.txt` ?

Reply:
@hsiang41  i see.. so what we need is to ready `grpc_tools.protoc` and `googleapis-common-protos` python modules for compiling the `.proto` definition file. since this grpc server is hosted in cloud, and the grpc service is only available to user who use this cloud service. i'd suggest package it downstream.